Garvan-Data-Science-Platform/ctrl
# [FIX] Prisma generated schema showing up in swagger

## Changes Made
- Script to replace all references of `DefaultSelection_Prisma`
- Run script when building docs
- Updated docs

## Test
- Run `yarn build-docs`
- Run `yarn dev` and check docs to see if `DefaultSelection_Prisma` is there.
